src/app/features/ledger/generic-steps/connect-device/connect-ledger-error.layout.tsx

```diff
@@ -54,36 +51,29 @@ export function ConnectLedgerErrorLayout(props: ConnectLedgerErrorLayoutProps) {
           Unable to connect
         </ErrorLabel>
       )}
-      <Stack
-        border="2px solid"
-        borderColor={color('border')}
-        borderRadius="12px"
-        my="loose"
-        mx="base"
-        spacing="base"
-        textAlign="left"
-        p="extra-loose"
-      >
+      <Stack borderRadius="12px" gap="space.01" textAlign="left" py="space.05">
         <PossibleReasonUnableToConnect text="Check if Ledger Live is open. Close it and try again" />
         <PossibleReasonUnableToConnect text="Ensure you only have one instance of Leather open" />
         <PossibleReasonUnableToConnect text="Verify the Stacks app is installed and open" />
         <PossibleReasonUnableToConnect text="Check you've approved the browser USB pop up" />
       </Stack>
-      <LeatherButton onClick={onTryAgain}>Try again</LeatherButton>
-      <Caption mt="loose">
+      <LeatherButton width="100%" onClick={onTryAgain}>
+        Try again
+      </LeatherButton>
+      <LCaption mt="loose">
         If the problem persists, check our{' '}
-        <Link
-          display="inline"
-          fontSize={1}
+        <LCaption
```

Comment:
no need to repeat the LCaption, could better just use a span?

Reply:
hm, why not?  it's a bit lazy to write these caption props all the time